tests/unit/Codeception/Module/PhpBrowserRestTest.php

```diff
@@ -51,7 +51,6 @@ public function testSendAbsoluteUrlGet()
     {
         $this->module->sendGET('http://127.0.0.1:8010/rest/user/');
         $this->module->seeResponseCodeIs(200);
-        $this->assertEquals('http://127.0.0.1:8010/rest/user/', $this->module->client->getHistory()->current()->getUri());
```

Comment:
REST module is not updating a history list. Not sure if this a feature or bug :)